OldStation (Beta/Charlie/Delta) QoL Improvements

Potato-Masher opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
I've been playing on the oldstation ghost role quite frequently lately and while I appreciate the recent update (#45838) to it, I've been slightly annoyed by a few things even still. Such as how there is no scrubber system (other than portable scrubbers) to deal with the miasma that builds up in the AI chamber from all the gibs, or how there is so much resin that has to be cleaned up which can take up a chunk of time just to roll around with a welder and a fuel tank burning away all the resin. Late joining can be a pain when you need to eat something but the weeds have consumed all remaining nutriment. Not only that, but I have also noticed a few instances of multiple resin floors on one turf, or even resin floors existing within walls. Plus I believe there is an air pipe missing leading to the mining bay.

So, I decided to take it upon myself to deal with this. The final product is this:

1. New revamped atmos. Before, it was simply a small room at the south end of Beta that contained an air mixer to distro and two small, sad N2 and Oxygen storage tanks. Not anymore, atmos is now over double the size and comes with an added basic scrubber system that covers the 3 stations. No filtering however, it just pumps waste directly into space. Note that atmos isn't functional at the start, survivors will still need to repair the missing pipes and vents that connect to the rest of the station for any of it to work. I've tested to make sure it works by flooding plasma at various points in the station and it seems to work fine once the air alarms are set to contaminated and the intended missing pipes are repaired.
Other atmos fixes include making all air alarms all access rather than just half of them, added missing pipes, and more visible vents.
**Current image of the new Beta Atmos:**
![tgstation_ _mapsRandomRuinsSpaceRuinsoldstation_2019-11-05_16-27-45](https://user-images.githubusercontent.com/33107541/68247642-45802680-ffe9-11e9-9216-1f0a087398f4.png)

2. Area improvement. Beta Station now has dedicated medbay and storage areas (with respective APC and air alarms) rather than all just being encompassed by the "North Hall" area. Other minor tweaks to make future editing a bit more easier to understand.

3. Delta improvements. I've fixed the weird unintentional extra resin tiles as well as making the resin infestation localized entirely inside the AI chamber. No need to spend extra time clearing RnD. Additionally an autolathe has been added.

4. Minor general improvements. External engineering access restrictions have been removed, and the airlock middle area has been expanded so less air is lost when fiddling around with crates (same as with the mining external airlock). Additional mask and air tanks added to mining to avoid having to run back to engineering to grab one. A crate with one of each of the fertilizers has been put in the corner of hydroponics. Trash cans and light switches added in rooms that would reasonably need one. I've thrown in a little additional metal and glass to some existing stacks, and some external decorations have been added as well as one fluff note.

This is my first mapping PR, I sure do hope I got the mapmerger working correctly.
**EDIT:** Success! I think.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Oldstation IMO was a little tedious when it came to cleaning out the resin, and the miasma was difficult to deal with with only one portable scrubber on Delta; these changes should reduce the amount of time dealt just clearing resin for those who feel the need to clear it and allow more time to build and repair the station to working order. Atmos was lacking as well, and is now equipped to deal with potential gas spills and miasma that may leak from the AI chamber (xenos will tend to open doors which allows the gas to get into the hallways).
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl: Potato Masher
add: Oldstation (ghost role spacestation) now has a proper basic atmos system with a scrubber loop, as well as additional N2 and O2 in enlarged tanks.
tweak: Oldstation QoL improvements, including reduced xeno resin spread and an added autolathe to RnD.
fix: Oldstation: A few missing pipes have been fixed, having multiple resin floors on one tile/resin floors inside walls have been fixed.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Potato-Masher committed:
Beta/Charlie/Delta station QoL improvements

Atmospherics is now actually a considerable part of Beta station with basic scrubber setup.

Missing atmos pipes added, air alarms are all now unlocked rather than just some, a slight increase in available metal and glass, removed access requirements on engineering external access airlocks, one extra fuel canister, and added light switches and trash cans due to the lack of disposals.

Delta: Added an autolathe to RnD, removed most instances of multiple resin floors on one turf/resin floors in walls, reduced resin spread to be just in the AI chamber, and one extra scrubber in the AI chamber to deal with the miasma buildup after long rounds.

Learner10 commented:
Hell yea. These are awesome QoL changes. I myself was personally bothered by that one piece of glowing resin that was buried beneath another ordinary looking piece of resin.

GunHog commented:
Is there any chance you would consider adding deactivated cameras around the station? If the players there decide to build an AI, one of them, or a borg/the AI's shell, would be much more likely to walk around the station fixing those up so it has something to see around the station other than just its own room.

Potato-Masher committed:
Minor changes to atmos and added a camera assembly crate in AI chamber

Moved the external filling station port pipes so they are not inside walls, fixed a missing floor decal. Added a camera assembly crate in the AI chamber for making a basic camera network in the event one would like to repair the AI.

Potato-Masher commented:
@GunHog Sounds fair, and I would do that but for the life of me I can't seem to find/figure out how to make a camera subtype that starts deactivated. The best I can do is this: I've thrown in an assembly crate in the AI chamber filled with cables and camera assemblies in the latest commit for those who wish to make use of the AI core already present in there.

I've also made some minor fixes in this commit, mainly a missing decal in atmos (you can see it in the example image) as well as moving the refilling station pipes out of the wall.
